src/main/java/network/brightspots/rcv/StreamingCvrReader.java

```diff
@@ -223,24 +226,34 @@ private void cvrCell(int col, String cellData) {
     // see if this column is in the ranking range
     if (col >= firstVoteColumnIndex
         && col < firstVoteColumnIndex + config.getMaxRankingsAllowed()) {
-
       int currentRank = col - firstVoteColumnIndex + 1;
       // handle any empty cells which may exist between this cell and any previous one
       handleEmptyCells(currentRank);
-      String candidate = cellData.trim();
-      // skip undervotes
-      if (!candidate.equals(config.getUndervoteLabel())) {
-        // map overvotes to our internal overvote string
-        if (candidate.equals(config.getOvervoteLabel())) {
-          candidate = Tabulator.EXPLICIT_OVERVOTE_LABEL;
-        } else if (!config.getCandidateCodeList().contains(candidate)
-            && !candidate.equals(config.getUndeclaredWriteInLabel())) {
-          // this is an unrecognized candidate so add it to the unrecognized candidate map
-          // this helps identify problems with CVRs
-          unrecognizedCandidateCounts.merge(candidate, 1, Integer::sum);
+      String cellString = cellData.trim();
+
+      // There may be multiple candidates in this cell (i.e. an overvote).
+      String[] candidates;
+      if (!isNullOrBlank(overvoteDelimiter)) {
+        candidates = cellString.split(overvoteDelimiter);
```

Comment:
Edge case that might cause a problem: if we have something like "A;" or ";A" where ; is the delimiter (I believe split would return "A" and "" for that, but not 100% sure).